src/Cli/dotnet/Commands/Tool/ToolCommandSpecCreator.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.CommandLine;
+using System.Text;
```

Comment:
nit: Most of the generic `System` usings should already be included automatically, thus redundant here.